flaskapp/models.py

```diff
@@ -7,13 +7,23 @@ class Question(db.Model):
     mark = db.Column(db.Integer, nullable=False)
     difficulty = db.Column(db.Integer, nullable=False)
     imp = db.Column(db.Boolean, default=False)
+    option1 = db.Column(db.Text,nullable=True,default=None)  #
```

Comment:
same here two models, use specialization here. mcq_question **is a** question kind of relation. checkout sqlalchemy for this. Maybe you can do this with backref